lib/licensee/projects/project.rb

```diff
@@ -8,16 +8,18 @@
 module Licensee
   module Projects
     class Project
-      attr_reader :detect_readme, :detect_packages
+      attr_reader :detect_readme, :detect_packages, :detect_reuse_licenses_dir
```

Comment:
Should this be an optional feature, or always on?

Reply:
Removed in the latest version, but I can restore if you want this to be an option.